packages/accounts/src/account.ts

```diff
@@ -37,6 +36,7 @@ import {
 import BN from 'bn.js';
 
 import { Connection } from './connection';
+import { TxOutcome } from '@near-js/types/lib/provider/response';
```

Comment:
Why isn't this imported like this?
```suggestion
import { TxOutcome } from '@near-js/types';
```

Reply:
This applies to all imports of `TxOutcome`

Reply:
it was automatically imported, forgot to export `TxOutcome` from `@near-js/types`, that's why it's `@near-js/types/lib/provider/response`. Will fix it.